spec/normalizy/filters/truncate_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Normalizy::Filters::Truncate do
```

Comment:
Still needs to test the `input.is_a?(String)` and  `options[:limit].is_a?(Integer)`.